lib/Command/FolderCommand.php

```diff
@@ -40,7 +40,14 @@ protected function getFolder(InputInterface $input, OutputInterface $output): ?a
 			return null;
 		}
 
-		$folder = $this->folderManager->getFolder($folderId, $this->rootFolder->getMountPoint()->getNumericStorageId());
+		$rootStorageId = $this->rootFolder->getMountPoint()->getNumericStorageId();
+		if ($rootStorageId === null) {
+			$output->writeln('<error>storage id not found ' . $rootStorageId . '</error>');
```

Comment:
You are concatenating with null here, not really useful.
```suggestion
			$output->writeln('<error>Root storage id not found</error>');
```